### ISSUE TITLE
[IxBreadcrumb] Uncaught TypeError: can't access property "tagName"

### ISSUE THREAD

**Issue by robert-wettstaedt** [siemens/ix#1143]
### Prerequisites

- [X] I have not leaked any internal company information.
- [X] I have not posted internal links or code snippets.

### What happened?

Using the `IxBreadcrumb` component will throw an uncaught exception: 

```
can't access property "tagName", element is null
```

### What type of frontend frameware are you seeing the problem on?

React

### Which version of iX do you use?

v2.1.1

### Code to produce this issue.

```shell
Codesandbox example: https://codesandbox.io/p/sandbox/ixbreadcrumb-cant-access-property-tagname-k6jjlg
```

**Comment by nuke-ellington** [comment]
@robert-wettstaedt should be fixed with `2.1.2-beta.0`. `2.1.2` will be released in the next few days.

**Comment by robert-wettstaedt** [comment]
Fixed in `2.1.2`
